### ISSUE TITLE
[LVGL] Add more one operation for each interrupt

### ISSUE THREAD

**Issue by elmonic** [eez-open/studio#481]
Hi, thank you
For version 15.1
In my opinion, it is better to be able to write more than one operation for each interrupt, though directly, not with the help of flow.
For example, I have two operations in RELEASED and in its function I can receive the pointer related to the widget directly, and this process cannot be done in flow mode with the command lv_event_get_target.
![Untitled](https://github.com/user-attachments/assets/e608d8d7-ff33-427f-93ec-3695790825b1)

**Comment by elmonic** [comment]
hi
What do you think about this problem?

**Comment by mvladic** [comment]
I don't understand what you mean, can you rephrase?

**Comment by elmonic** [comment]
I mean writing several actions for an event at the same time. This process can be done by Flow, but it is no longer possible to get the widget source inside the events (with the command lv_event_get_target). Unless a global variable is used that is defined.
